### PR TITLE
[ADD] Module which enable tracking history during import.

### PR DESCRIPTION
**Pull request by benwillig**

### CODE REVIEW

File: base_import_default_enable_tracking/static/src/xml/base_import.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates xml:space="preserve">
+
+    <t t-extend="ImportView">
+
+        <t t-jquery="input#oe_import_tracking" t-operation="replace">
+            <input type="checkbox" id="oe_import_tracking" checked="true"/>
```
Comment (yajo) [inline base_import_default_enable_tracking/static/src/xml/base_import.xml]: Valid HTML is `checked=""` or `checked="checked"`

File: base_import_default_enable_tracking/static/src/xml/base_import.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates xml:space="preserve">
+
+    <t t-extend="ImportView">
+
+        <t t-jquery="input#oe_import_tracking" t-operation="replace">
```
Comment (yajo) [inline base_import_default_enable_tracking/static/src/xml/base_import.xml]: Use `t-operation="attributes"`

File: base_import_default_enable_tracking/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': 'https://www.acsone.eu',
+    'depends': [
+        'base_import',
+    ],
+    'data': [
+    ],
+    'demo': [
+    ],
```
Comment (yajo) [inline base_import_default_enable_tracking/__manifest__.py]: Remove unneeded empty things.

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by benwillig** [comment]
@Yajo I updated my code. Thanks for the review.

File: base_import_default_enable_tracking/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Base Import Default Enable Tracking',
+    'summary': """
+        This modules simply enables history tracking when doing an import.""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': 'https://www.acsone.eu',
```
Comment (MiquelRForgeFlow) [inline base_import_default_enable_tracking/__manifest__.py:12]: `https://github.com/OCA/server-tools`

File: base_import_default_enable_tracking/README.rst [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+===================================
+Base Import Default Enable Tracking
+===================================
+
+This modules simply enables history tracking when doing an import.
+
+Installation
+============
+
+To install this module, you need click on install button.
```
Comment (MiquelRForgeFlow) [inline base_import_default_enable_tracking/README.rst:14]: You can delete installation and configuration sections

File: base_import_default_enable_tracking/__manifest__.py (hunk 4 above)
Comment (yajo) [reply base_import_default_enable_tracking/__manifest__.py:12]: No problem on letting their URL, they were the original authors.

**Review by yajo — APPROVED** [review]

File: base_import_default_enable_tracking/__manifest__.py (hunk 4 above)
Comment (MiquelRForgeFlow) [reply base_import_default_enable_tracking/__manifest__.py:12]: That's why I didn't do the "request changes"

File: base_import_default_enable_tracking/static/src/xml/base_import.xml [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+    <t t-extend="ImportView">
+
+        <t t-jquery="input#oe_import_tracking" t-operation="attributes">
+            <attribute name="checked">checked</attribute>
+        </t>
+
+    </t>
+
+</templates>
```
Comment (atchuthan) [inline base_import_default_enable_tracking/static/src/xml/base_import.xml:12]: empty line after `</template>`

**Review by atchuthan — APPROVED** [review]
code review :+1:

File: base_import_default_enable_tracking/README.rst [hunk 7]
```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================
+Base Import Default Enable Tracking
+===================================
+
+This modules simply enables history tracking when doing an import.
```
Comment (dreispt) [inline base_import_default_enable_tracking/README.rst:9]: This is a feature that for some reason was removed from the UI, right?
Any chance that it is is an upstream bug?

File: base_import_default_enable_tracking/__manifest__.py (hunk 4 above)
Comment (sbidoul) [reply base_import_default_enable_tracking/__manifest__.py:12]: Prefer https://github.com/OCA/server-tools, see also https://github.com/OCA/maintainer-tools/pull/276

File: base_import_default_enable_tracking/README.rst (hunk 7 above)
Comment (benwillig) [reply base_import_default_enable_tracking/README.rst:9]: This feature still exists but it's only available in debug mode. So it's not possible for a simple user to enable tracking before doing an import. If you install this module, tracking will always be enabled, and only a user who is in debug mode will be able to disable it.

Comment (dreispt) [reply base_import_default_enable_tracking/README.rst:9]: OK

**Review by dreispt — APPROVED** [review]

**Pull request merged**